bugalink-frontend/cypress/e2e/navigation.cy.ts

```diff
@@ -49,23 +49,23 @@ describe('Navigation tests', () => {
     cy.location('pathname').should('match', /^\/request\/[\w-]+\/accept$/);
   });
 
-  // /ride/<rideId>/detailsOne
+  // /ride/<rideId>/details
   it('should navigate to ride details one page', () => {
     cy.visit('/');
     cy.get('[data-cy="ride-details"]').first().click();
-    cy.location('pathname').should('match', /^\/ride\/[\w-]+\/detailsOne$/);
+    cy.location('pathname').should('match', /^\/ride\/[\w-]+\/details$/);
   });
 
   // /ride/<rideId>/detailsTwo
   it('should navigate to ride details two page', () => {
```

Comment:
This test should be changed such that we click on something like "PAGAR" and it takes us to the page for selecting payment methods for the ride

Reply:
Done